basic-javascript/project-1.spec.js

```diff
@@ -1,3 +1,182 @@
 const helpers = require('./project-1');
 
-// start testing!
+describe('multiplyByTen', () => {
+  test('Correct multiplication', () => {
+    const actual = helpers.multiplyByTen(2);
+    expect(actual).toEqual(20);
+  });
+  test('By NaN', () => {
+    const actual = helpers.multiplyByTen(NaN);
+    expect(actual).toEqual(NaN);
+  });
+  test('By a number in String format', () => {
+    const actual = helpers.multiplyByTen('2');
+    expect(actual).toEqual(20);
+  });
+  test('By a String other than a digit', () => {
+    const actual = helpers.multiplyByTen('hola');
+    expect(actual).toEqual(NaN);
+  });
+});
+describe('subtractFive', () => {
+  test('worng substraction', () => {
```

Comment:
Be careful of spelling errors, I recommend reading back over the code after you've finished just to catch little things like this. It doesn't affect how the code runs (usually) but spelling mistakes make it look unfinished.

Reply:
I'll be attentive -> I tend to mix some 'spanish' at writing. In spanigh subtract is 'substracción'.

Reply:
I know, and I'm not trying to make you feel bad. I make a lot of spelling mistakes when I write in other languages too. I just wanted to point it out so that when you start showing your code to employers you'll already be in the habit of double checking.

Reply:
Hello, your comments are always welcome and appreciated, that doesn't make me feel bad in any sense 😊.

I am aware that my English skills need a lot more work on, but my brain is kidnapped by Lambda curriculum 😅.